code/controllers/subsystem/job.dm

```diff
@@ -191,55 +191,55 @@ SUBSYSTEM_DEF(job)
 //This proc is called before the level loop of DivideOccupations() and will try to select a head, ignoring ALL non-head preferences for every level until
 //it locates a head or runs out of levels to check
 //This is basically to ensure that there's atleast a few heads in the round
-/datum/controller/subsystem/job/proc/FillHeadPosition()
-	for(var/level in level_order)
-		for(var/command_position in GLOB.command_positions)
-			var/datum/job/job = GetJob(command_position)
-			if(!job)
-				continue
-			if((job.current_positions >= job.total_positions) && job.total_positions != -1)
-				continue
-			var/list/candidates = FindOccupationCandidates(job, level)
-			if(!candidates.len)
-				continue
-			var/mob/dead/new_player/candidate = pick(candidates)
-			if(AssignRole(candidate, command_position))
-				return 1
-	return 0
+//datum/controller/subsystem/job/proc/FillHeadPosition()
+//	for(var/level in level_order)
+//		for(var/command_position in GLOB.command_positions)
+//			var/datum/job/job = GetJob(command_position)
+//			if(!job)
+//				continue
+//			if((job.current_positions >= job.total_positions) && job.total_positions != -1)
+//				continue
+//			var/list/candidates = FindOccupationCandidates(job, level)
+//			if(!candidates.len)
+//				continue
+//			var/mob/dead/new_player/candidate = pick(candidates)
+//			if(AssignRole(candidate, command_position))
+//				return 1
+//	return 0
```

Comment:
You can use /* and */ to start/close a comment across multiple lines respectively.
Make sure to add an austation note too